packages/flutter_tools/lib/src/commands/attach.dart

```diff
@@ -306,7 +307,9 @@ class AttachCommand extends FlutterCommand {
             fs.currentDirectory,
             LaunchMode.attach,
           );
+          flutterUsage.sendEvent('attach', 'success');
         } catch (error) {
+          flutterUsage.sendEvent('attach', 'failure');
```

Comment:
This information is only captured via IDE driven attach, you'll need to add a usage below outside of the daemon block.

Reply:
Thanks! Done.